### PR TITLE
Doc/update sentieon info

### PR DESCRIPTION
**Pull request by ashwini06**
### This PR:

 Address how to use `Sentieon` in balsamic documentation. 

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Comment by ashwini06** [comment]
doc1

**Review by khurrammaqbool — APPROVED** [review]
It may be nice to also add a step to install the latest changes from @develop branch.

File: docs/install.rst [hunk 1]
```diff
@@ -6,13 +6,20 @@ This section describes steps to install BALSAMIC (**version** = 4.1.0)
...
-Software Requirments
+Software Requirements
 ~~~~~~~~~~~~~~~~~~~~
 
-- Conda >=version 4.5.0: For detailed software and python requirments please see ``requirments.txt`` and ``BALSAMIC/conda/balsamic.yaml``
-
+- Conda >=version 4.5.0: For detailed software and python requirements please see ``setup.py`` and ``BALSAMIC/conda/balsamic.yaml``
 - Singularity >=version 3.0.0: BALSAMIC uses singularity to run vairous parts of the workflow. 
 - Python 3.6
+- Balsamic is dependent on third-party bioinformatics software ``Sentieon-tools``. Example: for running wgs variant calling using ``TNScope``, and to execute ``UMIworkflow``.
```
Comment (hassanfa) [inline docs/install.rst:15]:
```suggestion
- BALSAMIC is dependent on third-party bioinformatics software ``Sentieon-tools``. Example: for running wgs variant calling using ``TNScope``, and to execute ``UMIworkflow``.
```

**Review by hassanfa — COMMENTED** [review]
👍

**Review by hassanfa — APPROVED** [review]

**Comment by ashwini06** [comment]
> It may be nice to also add a step to install the latest changes from @develop branch.

@khurrammaqbool : Sorry, I didn't understand what you mean here.

**Pull request merged**